.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 2
+updates:
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
+    directory: "/"  # Location of package manifests
+    schedule:
+      interval: "daily"
+    ignore:
+      # support LTS 1.6.x containerd, ignore dependabot alerts for 1.7
+      - dependency-name: "github.com/containerd/containerd"
+        versions: ["1.7.x"]
```

Comment:
Is it necessary to ignore it here? Maybe we can let it cut PR but don't merge it?

Reply:
Update here. containerd 1.7.x has extended support now to align with containerd 1.6 LTS. So updates should be fine here.